src/main/java/com/robosh/ejournal/data/repository/impl/ValidationRepositoryImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.robosh.ejournal.data.repository.impl;
+
+import com.robosh.ejournal.data.repository.ValidationRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+import static com.robosh.ejournal.constant.Query.CHECK_EMAIL_UNIQUE;
+
+@Repository
+public class ValidationRepositoryImpl implements ValidationRepository {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Override
+    public boolean isUnique(String table, String column, String value) {
+        int result = entityManager
+                .createNativeQuery(CHECK_EMAIL_UNIQUE.getQuery(column, table, value))
+                .getFirstResult();
+        return result <= 0;
```

Comment:
does we need to check if it is less than zero?

Reply:
How can you transform int to boolean?

Reply:
just check if is equals 0

Reply:
it's something like caution. Ok, I'll change it if you want